nixos/doc/manual/development/option-types.xml

```diff
@@ -21,6 +21,16 @@
   </para>
 
   <variablelist>
+   <varlistentry>
+    <term>
+     <varname>types.anything</varname>
+    </term>
+    <listitem>
+     <para>
+      A type that accepts any value and merges attribute sets together.
```

Comment:
I would give some more explanation and examples here. At least one working and one failing example. Nothing is truly trivial for every reader :)

Reply:
Added an example now